pybaseball/plotting.py

```diff
@@ -1,34 +1,58 @@
-import altair as alt
-import pandas as pd
 from pathlib import Path
+from typing import Any, List, Optional
+
+import matplotlib.axes as axes
+import matplotlib.patches as patches
+import matplotlib.path
 import matplotlib.pyplot as plt
 import numpy as np
-from typing import Optional, Tuple
 import pandas as pd
 
-
 CUR_PATH = Path(__file__).resolve().parent
-STADIUM_COORDS = pd.read_csv(
-    Path(CUR_PATH, 'data', 'mlbstadiums.csv'), index_col=0
-)
+STADIUM_COORDS = pd.read_csv(Path(CUR_PATH, 'data', 'mlbstadiums.csv'), index_col=0)
+
 # transform over x axis
 STADIUM_COORDS['y'] *= -1
 
+# Use altair if available for backwards compatibility, else use pyplot
+try:
+    import altair as alt
+    import warnings
+
+    _ALTAIR_ENABLED = True
+    _ALTAIR_WARNING = """
+        Altair functionality is deprecated and will be removed in a future version.
+        Use pyplot functions instead or remove altair the altair library to prevent autodetection.
+    """
+except ImportError:
+    _ALTAIR_ENABLED = False
+
+    # Needed so we can still use altair datatypes for now in our func definitions
+    class FakeAltair():  # pylint: disable=too-few-public-methods
+        Chart = Any
+        LayerChart = Any
+
+    alt = FakeAltair()
```

Comment:
Do we _need_ this?

Reply:
I've cleaned this up so it's less ugly/hacky feeling.